source/Calamari.AzureAppService.Tests/AppServiceIntegrationTest.cs

```diff
@@ -115,12 +132,47 @@ protected static async Task DoWithRetries(int retries, Func<Task> action, int se
 
         protected void AddAzureVariables(CommandTestBuilderContext context)
         {
-            context.Variables.Add(AccountVariables.ClientId, clientId);
-            context.Variables.Add(AccountVariables.Password, clientSecret);
-            context.Variables.Add(AccountVariables.TenantId, tenantId);
-            context.Variables.Add(AccountVariables.SubscriptionId, subscriptionId);
-            context.Variables.Add("Octopus.Action.Azure.ResourceGroupName", resourceGroupName);
-            context.Variables.Add("Octopus.Action.Azure.WebAppName", site.Name);
+            AddAzureVariables(context.Variables);
+        }
+
+        protected void AddAzureVariables(VariableDictionary variables)
+        {
+            variables.Add(AccountVariables.ClientId, ClientId);
+            variables.Add(AccountVariables.Password, ClientSecret);
+            variables.Add(AccountVariables.TenantId, TenantId);
+            variables.Add(AccountVariables.SubscriptionId, SubscriptionId);
+            variables.Add("Octopus.Action.Azure.ResourceGroupName", ResourceGroupName);
+            variables.Add("Octopus.Action.Azure.WebAppName", WebSiteResource.Data.Name);
```

Comment:
These two magic strings look like they come from variables in the old `AzureWebAppHealthCheckActionHandlerFixtures.cs`. I'm wondering if we can use the variables here?

I think they are:
- `SpecialVariables.Action.Azure.ResourceGroupName`
- `SpecialVariables.Action.Azure.WebAppName`

Reply:
Yeah definitely. Let me fix that up

Reply:
Fixed in [9da81a6](https://github.com/OctopusDeploy/Calamari/pull/1091/commits/9da81a66f314946d5747ffbe45e601987a07237a)